R/family_tax_benefit.R

```diff
@@ -0,0 +1,129 @@
+#' FAMILY TAX BENEFIT
+#' 
+#' @param data Data table input. Each row is an individual. Column names are `HH_id`: household id, `id`: individual id, `age`: individual's age, `income`: individual's income, `in_secondary_school`: does the individual attend secondary school, `single_parent`: is the parent single, `other_allowance_benefit_or_pension`: does the individual receive a pension, benefit, or labour market program payment such as Youth Allowance, `maintenance_income`: the amount of maintenance income the individual receives for the care of a child/children from a previous relationship, `maintenance_children`: the number of children who you receive maintenance for that are in you care.
```

Comment:
Nothing wrong with `HH_id` but I've used `id_hh` in other scripts so can we use that instead please.